ami.tf

```
@@ -0,0 +1,45 @@
+
+locals {
+  need_ami_id = var.ami_image_id == null
+
+  // "amazon-eks-gpu-node-",
+  arch_label_map = {
+    "AL2_x86_64" : "",
+    "AL2_x86_64_GPU" : "-gpu",
+    "AL2_ARM_64" : "-arm64",
```

```suggestion
    "AL2_ARM_64" = "-arm64",
```

Although TF supports both ways (`=` and `:`), the canonical maps use `=`